src/static/js/language.js

```diff
@@ -0,0 +1,20 @@
+const languageSelect = document.querySelector('.language-switcher');
+languageSelect.addEventListener('change', (e) => {
+  if (e.target.value) {
```

Comment:
Does the change event ever fire when clicking on the default/selected option? If so we should add a check for that here because we don't want to reload the page and not change the current language.

Reply:
I've checked and the change event only fires when a different option is selected, so we should be good here :)